website/docs/d/database_pitr.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_database_point_in_time_recovery"
+description: |-
+  Get information about database_pitr
+subcategory: "Cloud Databases"
+---
+
+# ibm_database_pitr
```

Comment:
change this to the resource name "ibm_database_point_in_time_recovery"

Reply:
@hkantare thank you for catching that! :D